CONTRIBUTING.md

```diff
@@ -11,10 +11,11 @@ To add a dataset (see `datasets/yeast-osmotic-stress` as an example of a dataset
 1. Check that your dataset provider isn't already added (some of these datasets act as providers for multiple datasets)
 1. Create a new folder under `datasets/<your-dataset>`
 1. Add a `raw` folder containing your data
-1. Add an attached Snakefile that converts your `raw` data to `processed` data
-1. Add your snakefile to the top-level `run_snakemake.sh` file.
-1. If your dataset is a paper reproduction, add a `reproduction/raw` and `reproduction/processed` folder
+1. Add an attached Snakefile that converts your `raw` data to `processed` data.
+    - Make sure to use `uv` here. See `yeast-osmotic-stress`'s Snakefile for an example.
+1. Add your Snakefile to the top-level `run_snakemake.sh` file.
 1. Add your datasets to the appropiate `configs`
```

Comment:
This idea about just adding to the specific configs doesn't work when actually benchmarking. People will need to add a config for each dataset that has parameters tuned per algorithms

Reply:
I see. Do we have enough compute to do the paramater tuning server-side, or should we encode that information directly inside the configs?

Reply:
I would assume a user would need to do parameter tuning before adding it to the benchmarking repo/adding the config directly. We don't have a current automatic way to pick parameters (this is also the work I am working through to pick parameters/grid search so in the future we might have a parameter tuning server https://github.com/Reed-CompBio/spras/issues/206). We currently do not have a way to do cross dataset analysis (but I added an issue https://github.com/Reed-CompBio/spras/issues/205 about this idea a couple months ago)